modules/src/ics24_host/identifier.rs

```diff
@@ -59,7 +59,7 @@ impl ChainId {
         }
 
         let split: Vec<_> = chain_id.split('-').collect();
-        split[1].parse().unwrap_or(0)
+        split.last().expect("get revision number from chain_id").parse().unwrap_or(0)
```

Comment:
Would it be possible to have a small test for this code? A doc-test for the method `chain_version` could be ideal, just to show the expected behavior.

For example (this code is not tested):

```rust
    /// ```
    /// use ibc::ics24_host::identifier::ChainId;
    ///
    /// assert_eq!(ChainId::chain_version("chain-a-0"), 0);
    /// assert_eq!(ChainId::chain_version("ibc-10"), 10);
    /// assert_eq!(ChainId::chain_version("cosmos-hub---97"), 97);
    /// ```
    pub fn chain_version(chain_id: &str) -> u64 {
        ...
    }
```